### PR TITLE
NOISSUE - Update edge examples

### PR DESCRIPTION
**Pull request by SammyOina**
### What does this do?
updates examples to correct authorization

### Which issue(s) does this PR fix/relate to?
None

### List any changes that modify/break current functionality
No

### Have you included tests for your changes?
No

### Did you document any new/modified functionality?
No

### CODE REVIEW

File: docs/edge.md [hunk 1]
```diff
@@ -109,12 +109,12 @@ MF_AGENT_BOOTSTRAP_ID=34:e1:2d:e6:cf:03 ./mainflux-agent
...
-TH=`curl -s  -S -X GET http://some-domain-name:9013/things/bootstrap/34:e1:2d:e6:cf:03 -H "Authorization: edged" -H 'Content-Type: application/json' | jq -r .mainflux_id`
-KEY=`curl -s  -S -X GET http://some-domain-name:9013/things/bootstrap/34:e1:2d:e6:cf:03 -H "Authorization: edged" -H 'Content-Type: application/json' | jq -r .mainflux_key`
+CH=`curl -s -S -X GET http://some-domain-name:9013/things/bootstrap/34:e1:2d:e6:cf:03 -H "Authorization: Thing <BOOTSTRAP_KEY>" -H 'Content-Type: application/json' | jq -r '.mainflux_channels[0].id'`
+TH=`curl -s  -S -X GET http://some-domain-name:9013/things/bootstrap/34:e1:2d:e6:cf:03 -H "Authorization: Thing <BOOTSTRAP_KEY>" -H 'Content-Type: application/json' | jq -r .mainflux_id`
+KEY=`curl -s  -S -X GET http://some-domain-name:9013/things/bootstrap/34:e1:2d:e6:cf:03 -H "Authorization: Thing <BOOTSTRAP_KEY>" -H 'Content-Type: application/json' | jq -r .mainflux_key`
 
 # Subscribe for response
-mosquitto_sub -d -u $TH -P $KEY  -t channels/$CH/messages/res/# -h some-domain-name -p 1883
+mosquitto_sub -d -u $TH -P $KEY  -t 'channels/$CH/messages/res/#' -h some-domain-name -p 1883
```
Comment (rodneyosodo) [inline docs/edge.md]: It works without the quotations

Comment (SammyOina) [reply docs/edge.md]: on zsh it doesn't

Comment (rodneyosodo) [reply docs/edge.md]: I am using ZSH

image

**Review by drasko — APPROVED** [review]
LGTM

**Pull request merged**